Research/Research/RSDDesignSystem.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  RSDDesignSystem.swift
+//  Research
+//
+//  Copyright © 2019 Sage Bionetworks. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+//
+// 1.  Redistributions of source code must retain the above copyright notice, this
+// list of conditions and the following disclaimer.
+//
+// 2.  Redistributions in binary form must reproduce the above copyright notice,
+// this list of conditions and the following disclaimer in the documentation and/or
+// other materials provided with the distribution.
+//
+// 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+// may be used to endorse or promote products derived from this software without
+// specific prior written permission. No license is granted to the trademarks of
+// the copyright holders even if such marks are included in this software.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+import Foundation
+
+/// A task that implements this module is expected to include a pointer to the design system UI/UX rules to
+/// use that are specific to the tasks within this module.
```

Comment:
Is the thought that our design system rules should be the default for novel task modules?  We may want to mention that our design system rules are optimized more accessibility, usability, etc and that we recommend overriding it at the developers' peril

Reply:
In other words, may be worth mentioning a bit more about the optionality (or not) of using the default here, as it may be confusing for those not familiar with a design 'system' per se

Reply:
Absolutely, this should be looked upon as a "first pass" (and why this is version 0). For example, we also need to come up with a method for versioning views and view controllers.

That said, there is a balance between getting something in here but still be able to meet deadlines. Unfortunately, there comes a point where I have to get CRF out the door and don't have time to spend as much time as we'd all like writing documentation. :(

Reply:
WRT the thought, yes, there is a default design system and modules can (and probably should) include a version that the module should pin to for the rules used to develop that module.

Reply:
Excellent point. In order not to have the documentation need fall through the cracks, @DanWebster can you create a JIRA item about this in the Apple Client Frameworks (IA) project?

Reply:
On it